src/events/rawMessageReactionAdd.ts

```diff
@@ -30,7 +30,7 @@ export default class extends Event {
 			emoji: {
 				animated: raw.emoji.animated ?? false,
 				id: raw.emoji.id,
-				managed: 'managed' in raw.emoji ? raw.emoji.managed! : null,
+				managed: Reflect.get(raw.emoji, 'managed') || null,
```

Comment:
Get to 2020.
```suggestion
				managed: Reflect.get(raw.emoji, 'managed') ?? null,
```

Reply:
I saw the type was undefined or boolean so I thought maybe it has to catch off `false` and transform it to `null` as well :\